lib/twitter/enumerable.rb

```diff
@@ -2,16 +2,25 @@ module Twitter
   module Enumerable
     include ::Enumerable
 
+    METHODS = [:each, :each_page].freeze
```

Comment:
What purpose does this constant serve? I don’t see it used anywhere in the code. It seems like a case of [test-induced design damage](http://david.heinemeierhansson.com/2014/test-induced-design-damage.html). If it’s only used in the tests, this constant belongs in the tests, not in the code.